CHANGELOG.md

```diff
@@ -1,5 +1,97 @@
 # Changelog
 
+## 0.3.0
+
+This release stabilizes all preview functionality in uv. In brief, this means that:
+
+- `uv venv --python <version>` will [automatically
+  download](https://docs.astral.sh/uv/concepts/python-versions/#requesting-a-version) the Python
+  version if required
+- `uv venv` will read the required Python version from the `.python-version` file or
+  `pyproject.toml`
+- The uv [project](https://docs.astral.sh/uv/guides/projects/),
+  [tool](https://docs.astral.sh/uv/guides/tools/),
+  [script](https://docs.astral.sh/uv/guides/scripts/), and
+  [python](https://docs.astral.sh/uv/guides/install-python/) interfaces are considered stable and ready for
+  production-use.
+
+Additionally, we're excited to announce [official documentation](https://docs.astral.sh/uv/) for uv.
+
+### Breaking changes
+
+- Migrate to XDG and Linux strategy for macOS directories ([#5806](https://github.com/astral-sh/uv/pull/5806))
+- Move concurrency settings to top-level  ([#4257](https://github.com/astral-sh/uv/pull/4257))
+- Apply system Python filtering to executable name requests ([#4309](https://github.com/astral-sh/uv/pull/4309))
+- Remove `--legacy-setup-py` command-line argument ([#4255](https://github.com/astral-sh/uv/pull/4255))
+- Stabilize preview features ([#6166](https://github.com/astral-sh/uv/pull/6166))
+
+### Enhancements
```

Comment:
It's odd that these mismatch the actual big improvement we've done, we should give some indication that this changelog is different because all the actual enhancements are in the blog post and docs, maybe up in the into paragraph

Reply:
I don't think the breaking changes list being short is problematic. I mean, we can enumerate every new command but this is better done in the documentation.

Reply:
I meant it the other way round: Tell the user that the changelog below doesn't represent all the enhancement made (during the preview phase). Normally, i'd go to the changelog to learn about all the new features and changes i want to make, but for this release you should go to the blog post and then the docs. Something like saying "The changelog below includes only the changes made to the previously-preview features since the last release. The changes we made are too large to list them here, please refer to the blog post and the docs to learn about the new uv interface"

Reply:
Done. Will link to the blog when it's published in a fast-follow.